src/console/_style.scss

```diff
@@ -1,27 +0,0 @@
-.input-overlay-hack {
-    position: absolute;
-    height: 100%;
-    width: 100%;
-    opacity: 0;
-    top: 0;
-    left: 0;
-    cursor: pointer;
-  }
-
-.collapsible-trigger {
-  background-color: theme-color("primary", "background");
-}
```

Comment:
I don't see a replacement for this.

Reply:
@xitij2000 I did not see the collapsible trigger class being used inside of code.

Reply:
Yes, it is not used directly by code, but it's used by components that are used by this code indirectly. So you cannot remove it without a replacement.

Reply:
@xitij2000 
If I understand correctly, do you mean to say that the `collapsible-trigger` class is being used by components outside of this repo?

Inside this repo, as far as I can tell, the `collapsible-trigger` exists in test files.